src/hotspot/share/gc/shenandoah/shenandoahPhaseTimings.hpp

```diff
@@ -56,7 +56,7 @@ class outputStream;
   f(scan_roots,                                     "  Scan Roots")                    \
   SHENANDOAH_PAR_PHASE_DO(scan_,                    "    S: ", f)                      \
                                                                                        \
-  f(conc_mark_roots,                                "  Roots ")                        \
+  f(conc_mark_roots,                                "Concurrent Mark Roots ")          \
```

Comment:
This seems like fixing the regression from JDK-8255765, is it not? Should be a separate issue?